### ISSUE TITLE
filter cannot be saved in .sc files

### ISSUE THREAD

**Issue by amalgame21** [andmarti1424/sc-im#655]
filter cannot be saved in .sc files
After the filter is applied and the file is saved
Exit sc-im and reopen the file again
the filter is gone , instead those line was "filter" with :hiderow/hidecol 
but not :addfilter and :filteron

**Comment by andmarti1424** [comment]
yes, actually saving filters was not a thing we thought about when the filters were added.
it was more or less an instant filter and left the hidden rows/cols if someone wants to leave them hidden permanently.
move this to a discussion of feature request.
